Lesson 19 pop victor

VictorPop84 opened the pull request:
Homework

I also found : 

from contextlib import suppress

as an alternative.

VictorPop84 committed:
Lessson_19_HomeWork
KeyError and IndexError

Liitle commented:
> Homework
> 
> I also found :
> 
> from contextlib import suppress
> 
> as an alternative.

Yes that was the shortcut

Liitle commented on lesson_19/pop_victor/tema19-CM-VictorPOP.py:
```diff
@@ -0,0 +1,62 @@
+            return True
+        elif exc_type:
+            print('other error found with class')
+            return True
```
the other errors should propagate. If we return True the exceptions will be suppressed

VictorPop84 committed:
Lessson_19_HomeWork
a few improvements
replace False with True
also fixed the return on errors with functions
